build.gradle

```diff
@@ -5,10 +5,11 @@
  */
 buildscript {
     repositories {
+        google()
```

Comment:
Is this extra repo really needed?

Reply:
It's required for Android Gradle plugin 3.0.0